Package.swift

```diff
@@ -17,7 +17,7 @@ import PackageDescription
 let package = Package(
     name: "swift-openapi-runtime",
     platforms: [
-        .macOS(.v13), .iOS(.v16), .tvOS(.v16), .watchOS(.v9),
+        .macOS(.v10_15), .iOS(.v12), .tvOS(.v13), .watchOS(.v6),
```

Comment:
```suggestion
        .macOS(.v10_15), .iOS(.v13), .tvOS(.v13), .watchOS(.v6),
```

Let's keep the OS versions aligned.